plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -90,6 +101,9 @@ static void *ffmpeg_mux_create(obs_data_t *settings, obs_output_t *output)
 	signal_handler_t *sh = obs_output_get_signal_handler(output);
 	signal_handler_add(sh, "void file_changed(string next_file)");
 
+	proc_handler_t *ph = obs_output_get_proc_handler(output);
+	proc_handler_add(ph, "void split_file()", split_file_proc, stream);
```

Comment:
```suggestion
	proc_handler_add(ph, "void split_file(out bool split_file_enabled)", split_file_proc, stream);
```
Having `out bool split_file_enabled` in the parameter list looks the standard in this project.